diff --git a/node_modules/react-native-safe-area-context/android/.gradle/8.10/checksums/checksums.lock b/node_modules/react-native-safe-area-context/android/.gradle/8.10/checksums/checksums.lock
new file mode 100644
index 0000000..2d49ca1
Binary files /dev/null and b/node_modules/react-native-safe-area-context/android/.gradle/8.10/checksums/checksums.lock differ
diff --git a/node_modules/react-native-safe-area-context/android/.gradle/8.10/checksums/sha1-checksums.bin b/node_modules/react-native-safe-area-context/android/.gradle/8.10/checksums/sha1-checksums.bin
new file mode 100644
index 0000000..5be01df
Binary files /dev/null and b/node_modules/react-native-safe-area-context/android/.gradle/8.10/checksums/sha1-checksums.bin differ
diff --git a/node_modules/react-native-safe-area-context/android/.gradle/8.10/dependencies-accessors/gc.properties b/node_modules/react-native-safe-area-context/android/.gradle/8.10/dependencies-accessors/gc.properties
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-safe-area-context/android/.gradle/8.10/fileChanges/last-build.bin b/node_modules/react-native-safe-area-context/android/.gradle/8.10/fileChanges/last-build.bin
new file mode 100644
index 0000000..f76dd23
Binary files /dev/null and b/node_modules/react-native-safe-area-context/android/.gradle/8.10/fileChanges/last-build.bin differ
diff --git a/node_modules/react-native-safe-area-context/android/.gradle/8.10/fileHashes/fileHashes.bin b/node_modules/react-native-safe-area-context/android/.gradle/8.10/fileHashes/fileHashes.bin
new file mode 100644
index 0000000..4bfa3e4
Binary files /dev/null and b/node_modules/react-native-safe-area-context/android/.gradle/8.10/fileHashes/fileHashes.bin differ
diff --git a/node_modules/react-native-safe-area-context/android/.gradle/8.10/fileHashes/fileHashes.lock b/node_modules/react-native-safe-area-context/android/.gradle/8.10/fileHashes/fileHashes.lock
new file mode 100644
index 0000000..19b9933
Binary files /dev/null and b/node_modules/react-native-safe-area-context/android/.gradle/8.10/fileHashes/fileHashes.lock differ
diff --git a/node_modules/react-native-safe-area-context/android/.gradle/8.10/gc.properties b/node_modules/react-native-safe-area-context/android/.gradle/8.10/gc.properties
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-safe-area-context/android/.gradle/buildOutputCleanup/buildOutputCleanup.lock b/node_modules/react-native-safe-area-context/android/.gradle/buildOutputCleanup/buildOutputCleanup.lock
new file mode 100644
index 0000000..e4be38f
Binary files /dev/null and b/node_modules/react-native-safe-area-context/android/.gradle/buildOutputCleanup/buildOutputCleanup.lock differ
diff --git a/node_modules/react-native-safe-area-context/android/.gradle/buildOutputCleanup/cache.properties b/node_modules/react-native-safe-area-context/android/.gradle/buildOutputCleanup/cache.properties
new file mode 100644
index 0000000..c2ec388
--- /dev/null
+++ b/node_modules/react-native-safe-area-context/android/.gradle/buildOutputCleanup/cache.properties
@@ -0,0 +1,2 @@
+#Sat Jan 18 23:07:41 WIB 2025
+gradle.version=8.10
diff --git a/node_modules/react-native-safe-area-context/android/.gradle/nb-cache/android-1710055590/project-info.ser b/node_modules/react-native-safe-area-context/android/.gradle/nb-cache/android-1710055590/project-info.ser
new file mode 100644
index 0000000..cb3926b
Binary files /dev/null and b/node_modules/react-native-safe-area-context/android/.gradle/nb-cache/android-1710055590/project-info.ser differ
diff --git a/node_modules/react-native-safe-area-context/android/.gradle/nb-cache/subprojects.ser b/node_modules/react-native-safe-area-context/android/.gradle/nb-cache/subprojects.ser
new file mode 100644
index 0000000..9254355
Binary files /dev/null and b/node_modules/react-native-safe-area-context/android/.gradle/nb-cache/subprojects.ser differ
diff --git a/node_modules/react-native-safe-area-context/android/.gradle/nb-cache/trust/054836E91EB9D1F65E4432288EB13D9FADF0EE4FABF8C8B7BB487070BE069090 b/node_modules/react-native-safe-area-context/android/.gradle/nb-cache/trust/054836E91EB9D1F65E4432288EB13D9FADF0EE4FABF8C8B7BB487070BE069090
new file mode 100644
index 0000000..e3126df
--- /dev/null
+++ b/node_modules/react-native-safe-area-context/android/.gradle/nb-cache/trust/054836E91EB9D1F65E4432288EB13D9FADF0EE4FABF8C8B7BB487070BE069090
@@ -0,0 +1 @@
+7A818207242E296968D4ADE62BACF78789F56DD308C61C8DF6D8BED39CA4B0B0
diff --git a/node_modules/react-native-safe-area-context/android/.gradle/vcs-1/gc.properties b/node_modules/react-native-safe-area-context/android/.gradle/vcs-1/gc.properties
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-safe-area-context/android/build.gradle b/node_modules/react-native-safe-area-context/android/build.gradle
index e556043..d9ffac5 100644
--- a/node_modules/react-native-safe-area-context/android/build.gradle
+++ b/node_modules/react-native-safe-area-context/android/build.gradle
@@ -39,6 +39,10 @@ if (isNewArchitectureEnabled()) {
 }
 
 android {
+    buildFeatures {
+        buildConfig true
+    }
+    namespace "com.th3rdwave.safeareacontext"
     def agpVersion = com.android.Version.ANDROID_GRADLE_PLUGIN_VERSION
     if (agpVersion.tokenize('.')[0].toInteger() >= 7) {
         namespace "com.th3rdwave.safeareacontext"
